std_msgs/Header header
sensor_msgs/Image image
geometry_msgs/Point[] object_positions